topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -756,6 +889,76 @@ We now run **featureCounts** to count the number of reads per annotated gene.
+
+> ### {% icon hands_on %} Hands-on: Reformatting STAR output
+>
+> 1. Inspect the counts from `GSM461177_untreat_paired` in the collection `RNA STAR on collection N: reads per gene`
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many reads are unmapped/multi-mapped?
+> 2. At which line starts gene counts?
+> 3. What are the different columns?
+> 4. Which columns are the most interesting for our dataset?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. There are 1,190,029 unmapped reads and 571,324 multi-mapped reads.
+> > 2. It starts at line 5 with the gene `FBgn0085804`.
+> > 3. There are 4 columns:
+> >    1. Gene ID
+> >    2. Counts for unstranded RNA-seq
+> >    3. Counts for the 1st read strand aligned with RNA
+> >    4. Counts for the 2nd read strand aligned with RNA
+> > 4. We need the Gene ID column and the 2nd column because of the unstrandness of our data
+> >
+> {: .solution}
+>
+{: .question}
+
+We will reformat the output of **STAR** to be similar to the output of **featureCounts** (or other counting softwares) which is only 2 columns, one with IDs and the other one with counts.
+
+> ### {% icon hands_on %} Hands-on: Reformatting STAR output
+>
+> 1. {% tool [Select last](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_tail_tool/1.1.0) %} lines from a dataset (tail) to remove the first 4 lines with the following parameters:
+>    - {% icon param-collection %} *"Text file"*: `RNA STAR on collection N: reads per gene` (output of **RNA STAR** {% icon tool %})
+>    - *"Operation"*: `Keep everything from this line on`
+>    - *"Number of lines"*: `5`
+>
+> 2. {% tool [Cut](Cut1) %} columns from a table with the following parameters:
+>    - *"Cut columns"*: `c1,c2`
+>    - *"Delimited by"*: `Tab`
+>    - {% icon param-collection %} *"From"*: `Select last on collection N` (output of the **Select last** {% icon tool %})
+>
+> 3. Rename the collection `FeatureCount-like files`
+{: .hands_on}
+
+Later on the tutorial we will need to get the size of each gene. This is one of the output of **FeatureCounts** but we can also obtain it directly from the gene annotation file. As this is quite long, we recommand to launch it now.
+
+> ### {% icon hands_on %} Hands-on: Getting gene length
+>
+> 1. {% tool [Gene length and GC content](toolshed.g2.bx.psu.edu/repos/iuc/length_and_gc_content/length_and_gc_content/0.1.2) %} with the following parameters:
+>    - *"Select a built-in GTF file or one from your history"*: `Use a GTF from history`
+>      - {% icon param-file %} *"Select a GTF file"*: `Drosophila_melanogaster.BDGP6.87.gtf`
+>    - *"Analysis to perform"*: `gene lengths only`
+>
+>    > > ### {% icon warning %} Check the version of the tool below.
```

Comment:
```suggestion
>    > ### {% icon warning %} Check the version of the tool below.
```

Reply:
It doesn't render correctly; I think that it fixes the problem.

![Screenshot from 2022-04-14 12-30-55](https://user-images.githubusercontent.com/26678117/163371047-4acb968e-16fd-4ce7-8834-04c152a0ceb8.png)

Reply:
Thanks I had found it but forgot to push.